### ISSUE TITLE
wrfrst file cannot be generated

### ISSUE THREAD

**Issue by smileMchen** [wrf-model/WRF#1849]
I am running some test cases and find that wrfrst files cannot be generated in some cases. 
This is not always the case and the problem seems to happen randomly. 
The job didn't stop or get killed, but just hang there.

**Comment by weiwangncar** [comment]
@smileMchen Are you using the release v4.5 code?

**Comment by smileMchen** [comment]
@weiwangncar 
yes this is the V45 code. This issue occurs more frequently when running nest case.

**Comment by smileMchen** [comment]
@weiwangncar 
Wei, I repeated a user's case but the case was deleted.

**Comment by weiwangncar** [comment]
@smileMchen If you are still helping this user, have him/her use more processors to run - that way each processor will have access to more memory.

**Comment by smileMchen** [comment]
yes that is one option. I did see another user reported the same issue and fixed it by running with more processors. Thanks.
